### ISSUE TITLE
Thank you 

### ISSUE THREAD

**Issue by Wehrmicel** [Tharthar2/SaF2021#1]
Thank you for your amazing work ! I'm so happy to see that this mod is finally being updated !

May I know if you still plan to release this on curse?

Thanks !

**Comment by minhtet-htoon** [comment]
I’m ready to release it but since this is an unauthorized port, I’m waiting for the original dev to give me permission to relax it. Unfortunately, I’ve sent numerous emails but have not gotten a response.

**Comment by minhtet-htoon** [comment]
Err, release not relax. iPhone autocorrect is super annoying sometimes.

**Comment by Wehrmicel** [comment]
If I understand correctly, they are okay with it since they answered this regarding the update:
"probably not anytime soon, but another person might try and pick it up"

https://github.com/ChaosBuffalo/SpartanFire/issues/8

**Comment by minhtet-htoon** [comment]
﻿I’m trying to license their textures. From my limited legal understanding, something like the MIT license only applies to code and assets are usually ARR.
When I asked on that thread, one of the secondary devs pinged the main one saying they didn't know if I could use the textures.

Sent from my iPhone

**Comment by XxInvictus** [comment]
Just an FYI, I have taken on the work of performing a complete update for 1.16.5 with Kovak providing some assistance, here is the Git I am working out of and the current PR within Kovak's repo. I can assure you Kovak is still around and active on other projects. :)

You can see the commit history for a summary on what I have done, and what is outstanding if you wish to assist!

My Fork: https://github.com/XxInvictus/SpartanFire/tree/1.16.x
PR in Original Repo: https://github.com/ChaosBuffalo/SpartanFire/pull/12
1.16.x Branch Commit History Direct Link: https://github.com/XxInvictus/SpartanFire/commits/1.16.x

**Comment by XxInvictus** [comment]
TLDR:
For the most part the 1.16.5 update is working, it just requires some fixes and textures before it would be production ready.

**Comment by Red6785** [comment]
Do you plan to release it on curseforge, or will it stay as a github thing?

**Comment by minhtet-htoon** [comment]
I'm honestly not sure of the fate of this project. I did this port for a mod pack which was for a SMP server but the project never got off the ground due to issues with hosting so if there is public interest for it, yes but don't expect me do it atm. my school semester starts in 3 days so I doubt I will have any time to do so.

**Comment by Red6785** [comment]
Ah, alright. I've had an issue with this and other mods where it breaks when I try to build it and I don't know why. I have java and run gradlew build in the root but the build fails.

**Comment by Pugnatir** [comment]
Still here?

**Comment by minhtet-htoon** [comment]
I’m unfortunately unable to continue this project. If anyone is willing to take it on, I’ll transfer ownership to you.

**Comment by denispionicul** [comment]
someone could possibly release this on curseforge

**Comment by Pugnatir** [comment]
Hello im back and i have 1 little problem with some items when i'm attacking enemy with some items my game crashes.
If i know how to continiue it i can but i dont have knowledge how to do it
